.github/workflows/merge-prometheus-operator.yaml

```diff
@@ -27,6 +28,8 @@ jobs:
         example
         go.mod
         go.sum
+        scripts/go.mod
```

Comment:
I'd like to know more about why those are needed now

Reply:
I tried manually to sync and it there were merge conflicts due to the addition of `toolchain go1.23.3` . It should have been added previously too but we don't get much conflict on this so never needed.

https://github.com/prometheus-operator/prometheus-operator/blob/main/scripts/go.mod#L5
https://github.com/openshift/prometheus-operator/blob/main/scripts/go.mod#L5

Reply:
See from recent run https://github.com/rhobs/syncbot/actions/runs/13153490415/job/36705315910 as well